### PR TITLE
LRAC-14587 follow up unit tests

### PR DESCRIPTION
**Pull request by interaminense**
JIRA Issue https://issues.liferay.com/browse/LRAC-14587

### CODE REVIEW

**Comment by interaminense** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx [hunk 1]
```diff
@@ -106,13 +106,13 @@ describe('View Channel', () => {
 			)
 		);
 
-		const {container} = render(<DefaultComponent />);
+		const {container, debug} = render(<DefaultComponent />);
```
Comment (fmarcio) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx:109]:
```suggestion
		const {container} = render(<DefaultComponent />);
```

hey @interaminense please remove this "debug", ok? Thanks!

**Review by fmarcio — CHANGES_REQUESTED** [review]
hey @interaminense could you please remove this "debug". Thanks

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx (hunk 1 above)
Comment (interaminense) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx]: Done. Thanks!

**Comment by interaminense** [comment]
ci:forward

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**